### PR TITLE
PW-300 Hosted CSE solution, change from CSE key to library token

### PR DESCRIPTION
**Pull request by rikterbeek**
**Description**

Move to hosted CSE solution

**Tested scenarios**

Tested creditcard, stored payment method with CVC and backend orders

### CODE REVIEW

**Comment by rikterbeek** [comment]
if libraryToken if empty to fill in the libraryToken

File: view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js [hunk 1]
```diff
@@ -69,21 +68,29 @@ define(
...
-            initialize: function() {
+            initialize: function () {
                 var self = this;
                 this._super();
 
+                debugger;
+                ;
+                // include dynamic cse javascript
+                var dfScriptTag = document.createElement('script');
+                dfScriptTag.src = "//test.adyen.com/hpp/cse/js/" + this.getLibraryToken() + ".shtml";
```
Comment (Aleffio) [inline view/frontend/web/js/view/payment/method-renderer/adyen-cc-method.js]: dynamically choose test/live

File: view/adminhtml/templates/form/cc.phtml [hunk 2]
```diff
@@ -87,60 +100,72 @@ $ccExpYear = $block->getInfoData('cc_exp_year');
...
+        require(
+            [
+                'jquery',
+                'Magento_Sales/order/create/scripts',
+                "Magento_Sales/order/create/form"
+            ],
+            function () {
+                //<![CDATA[
+                var dfScriptTag = document.createElement('script');
+                dfScriptTag.src = "//test.adyen.com/hpp/cse/js/" + <?php echo $block->getLibraryToken(); ?> +".shtml";
```
Comment (Aleffio) [inline view/adminhtml/templates/form/cc.phtml]: dynamically choose test/live

File: view/frontend/web/js/view/payment/method-renderer/adyen-oneclick-method.js [hunk 3]
```diff
@@ -56,32 +55,42 @@ define(
                     ]);
                 return this;
             },
+            initialize: function () {
+                var self = this;
+                this._super();
+
+                // include dynamic cse javascript
+                var dfScriptTag = document.createElement('script');
+                dfScriptTag.src = "//test.adyen.com/hpp/cse/js/" + this.getLibraryToken() + ".shtml";
```
Comment (Aleffio) [inline view/frontend/web/js/view/payment/method-renderer/adyen-oneclick-method.js]: dynamically choose test/live

**Pull request merged**